concreate/descriptor.py

```diff
@@ -25,6 +24,8 @@ def __init__(self, descriptor_path, descriptor_type):
         if descriptor_type == 'image':
             self.check_schema_version()
 
+        self.module_repositories = {}
```

Comment:
This doesn't belong to Descriptor class - its not it responsibility to take care repositories implementations - move it to better place (maybe Generator or Repositories class injected to Generator later? )

Reply:
Will rewrite this.